src/components/views/read/cvForm/Education.tsx

```diff
@@ -126,206 +155,165 @@ const Education: FC<Props> = ({
     { key: 'gpa', label: 'GPA' },
     { key: 'country', label: 'Country' },
   ];
-  const handleNext = (): void => {
-    const modifiedValues = cards.map((card) => {
-      const cardInputValues = inputValues.find(
-        (inputValue) => inputValue.cardId === card.id,
-      );
-
-      return {
-        Degree: cardInputValues?.values.degree || '',
-        'Institution Name': cardInputValues?.values.instName || '',
-        Major: cardInputValues?.values.major || '',
-        'Start Date': cardInputValues?.values.startDate || '',
-        'End Date': cardInputValues?.values.endDate || '',
-        GPA: cardInputValues?.values.gpa || '',
-        Country: cardInputValues?.values.country || '',
-      };
-    });
-
-    handleValues('Education', modifiedValues);
-    nextPage();
-    nextStep();
-  };
-  const handleChangeSecondary = (field: string, value: string): void => {
-    setInputValuesSecondary((prevInputValues) => {
-      const updatedInputValues = [...prevInputValues];
-      const fieldValue = { [field]: value };
-      const existingFieldIndex = updatedInputValues.findIndex(
-        (inputValue) => Object.keys(inputValue)[0] === field,
-      );
-      if (existingFieldIndex !== -1) {
-        // Field already exists, update its value
-        updatedInputValues[existingFieldIndex] = fieldValue;
-      } else {
-        // Field doesn't exist, add it to the input values
-        updatedInputValues.push(fieldValue);
-      }
-      console.log(updatedInputValues);
-      return updatedInputValues;
-    });
-  };
   return (
-    <div>
-      <h2>Education</h2>
-      <div>
-        {cards.map((card, index) => (
+    <Box>
+      <Box>
+        {educationCards.map((card) => (
           <Card key={card.id}>
             <CardContent>
-              <Typography gutterBottom variant="h5" component="div">
+              <Typography gutterBottom variant="h5">
```

Comment:
TODO: new issue.

Maybe put the card in a separate component that would handle its internal state. For example, the isPresent, showFields etc...